openshift/message-board/extra/message-server-task/src/main/java/it/redhat/demo/task/UpdateUserBoardTask.java

```diff
@@ -18,52 +19,58 @@
 import it.redhat.demo.model.Message;
 import it.redhat.demo.model.MessageId;
 
-public class UpdateUserBoardTask implements ServerTask<Void> {
-
-	private static final String MESSAGE_ID_PARAM_NAME = "messageId";
+public class UpdateUserBoardTask implements ServerTask<Integer> {
```

Comment:
What's the Integer?

Reply:
The size of the board after we've added the message. I will add that on a javadoc.